doc/advanced/content/pcl_style_guide.rst

```diff
@@ -323,6 +323,40 @@ Command line usage:
    $ make format
 
 
+2.7. Includes
+^^^^^^^^^^^^^
+
+For consistent usage, headers should be included in the following order with alphabetical grouping ensured:
+
+1.  PCL headers
+
+    i.  All modular PCL includes, except main includes of common module
+    #.  The main PCL includes of common module
```

Comment:
The concept of what is a "main include of common module" is not clear. I had to go to the clang-format file to understand what it represented. Provide examples for what a "normal modular include" and a "main include of a common module" are.

Reply:
Looks good? 1f52a71